docs/docs/20-quickstart.md

```diff
@@ -482,6 +488,18 @@ the previous section.
     export FREIGHT_ALIAS=$(kargo get freight --project kargo-demo --output jsonpath={.alias})
     ```
 
+   For a comprehensive overview of your `Project`, follow these steps to access and 
+   view the components on the [Kargo Dashboard](https://localhost:31444/):
+
+   1. Navigate to the Kargo Dashboard by clicking the link above or entering the URL directly in your browser.
+   1. In the left-hand menu, select the <Hlt>Projects</Hlt> section.
+   1. Choose your `Project`, <Hlt>kargo-demo</Hlt>, from the list of available `Project`s.
+   1. Once there, you will see a detailed visual representation of the `Project`. This includes:
+      1. The <Hlt>Warehouse</Hlt> and its status.
+      1. Various `Stage`s: <Hlt>test</Hlt>, <Hlt>uat</Hlt>, and <Hlt>prod</Hlt>.
+
+      ![Argo-dashboard-screenshot](../static/img/kargo-dashboard-projects.png)
+
```

Comment:
You have this positioned in a section titled "Hands on with the Kargo CLI," which makes this feel somewhat misplaced, but we can fix that...

What I suggest is breaking up the whole "Hands on with the Kargo CLI" section into two separate sections for "Define Your First Pipeline" and "Promote to Test."

"Define Your First Pipeline" should probably remain mostly (but not entirely) CLI-centric for the simple reason that it involves a lot of YAML and environment variables and UI-based instructions are going to make this task harder rather than easier. This tracks with expected/observed usage. Since pipelines are defined declaratively, people tend to create them with `kargo apply`, `kubectl apply`, or via gitops. I think you could even introduce the section by _saying_ that the CLI is the most convenient way to define the pipeline. That said, I believe that once you get past `kargo apply`, you can probably swap over to UI-based instructions. Instead of inviting users to `kargo get freight` or `kargo get stages`, you can walk them through using the UI to see the pipeline they just created, witness the first piece of Freight popping up in the Freight timeline, etc.

"Promote to Test" being pretty much exclusively about imperative actions (and not declarative configuration), you could just cut over, completely to UI-based instructions.

Anticipating a question about whether this means few CLI examples in the docs overall -- in the short term, it does. But I believe detailed usage of CLI commands will reappear over time in individual guides. Let's keep the focus of the quickstart on imparting a maximum amount of knowledge with minimal reader effort.